### PR TITLE
Added Full-Screen Toggle Feature

### PR DESCRIPTION
**Pull request by sirjan13**
Added an option to toggle Full-Screen mode.
It also changes the title attribute when toggling modes.
(The full-screen mode is Similar to Code-Chef's ide Full Screen mode )

## **changes proposed :**

In Normal Mode:
image
 
In Full Screen Mode:
image

### CODE REVIEW

**Comment by omerjerk** [comment]
@sirjan13 This looks good. Please fix the merge conflicts.

File: css/stylesheet.css [hunk 1]
```diff
@@ -1608,3 +1609,40 @@ footer.bg-dark a {
...
+.panel-actions a {
+  color:#333;
+}
+
+.fsHide.fs {
+	height : 0px;
+	display :none;
+}
+
+.panel-fullscreen {
```
Comment (prateek27) [inline css/stylesheet.css]: Add Proper Indendation here.

Comment (sirjan13) [inline css/stylesheet.css]: Please review the changes in #31 .I have fixed the indentation.

**Comment by prateek27** [comment]
@sirjan13 can you resolve the conflicts in this. Looking good to me too.

**Comment by sirjan13** [comment]
@prateek27 fixed... #40

**Comment by championswimmer** [comment]
@sirjan13 there are some merge conflicts. can you resolve them please

**Comment by sirjan13** [comment]
@championswimmer the PR #40 which resolved everything was merged successfully

**Comment by championswimmer** [comment]
ok closing this then

**Pull request closed without merging**